integration_tests/snapshots/logs/async-metrics_ruby32.log

```diff
@@ -23,3 +11,21 @@ W, [XXXX]  WARN XXXX[datadog] Unable to patch Datadog::Tracing::Contrib::Aws::In
 {"traces":[[{"error":0,"meta":{"XXXX": "XXXX"},"metrics":{"XXXX": "XXXX"},"name":"aws.lambda","parent_id":"XXXX","resource":"dd-tracer-serverless-span","service":"aws.lambda","span_id":"XXXX","trace_id":"XXXX","type":"serverless","span_links":[],"start":XXXX,"duration":XXXX}]]}
 {"traces":[[{"error":0,"meta":{"XXXX": "XXXX"},"metrics":{"XXXX": "XXXX"},"name":"aws.lambda","parent_id":"XXXX","resource":"dd-tracer-serverless-span","service":"aws.lambda","span_id":"XXXX","trace_id":"XXXX","type":"serverless","span_links":[],"start":XXXX,"duration":XXXX}]]}
 {"traces":[[{"error":0,"meta":{"XXXX": "XXXX"},"metrics":{"XXXX": "XXXX"},"name":"aws.lambda","parent_id":"XXXX","resource":"dd-tracer-serverless-span","service":"aws.lambda","span_id":"XXXX","trace_id":"XXXX","type":"serverless","span_links":[],"start":XXXX,"duration":XXXX}]]}
+/opt/ruby/gems/3.2.0/gems/datadog-2.12.0/lib/datadog/core/logger.rb:14: info: Datadog::Core::Logger#initialize: does not call super probably
+/opt/ruby/gems/3.2.0/gems/datadog-2.12.0/lib/datadog/core/logger.rb:14: info: Datadog::Core::Logger#initialize: does not call super probably
+/opt/ruby/gems/3.2.0/gems/logger-1.7.0/lib/logger.rb:482: warning: Logger not initialized properly
+/opt/ruby/gems/3.2.0/gems/logger-1.7.0/lib/logger.rb:482: warning: Logger not initialized properly
+/opt/ruby/gems/3.2.0/gems/logger-1.7.0/lib/logger.rb:677: warning: Logger not initialized properly
+/var/runtime/lib/runtime.rb:26: info: LoggerPatch#initialize: does not call super probably
```

Comment:
This doesn't look right

Reply:
Looks like it might be related to https://github.com/aws/aws-lambda-ruby-runtime-interface-client/issues/33 ?